tests/Functional/Schema/SchemaManagerFunctionalTestCase.php

```diff
@@ -579,6 +579,32 @@ public function testListForeignKeys(): void
         self::assertEquals('CASCADE', $fkeys[0]->getOption('onDelete'));
     }
 
+    public function testCreateForeignKeyGh5485(): void
```

Comment:
Please refrain from using issue numbers in test names. Instead, describe what case the test covers.

Reply:
done